test/tracker/sample_roi.jl

```diff
@@ -0,0 +1,28 @@
+using Images
+using ImageTracking:sample_roi
+
+@testset "ROI sampling" begin
+    @info "Running ROI sampling tests"
+    box = BoxROI(ones(300, 300), [125, 125, 175, 175])
```

Comment:
To make it less error-prone, I think you need to cover the following types for`box.img`:

```julia
img = ones(300, 300)
testcolors = (RGB,Gray)
testtypes =  (Float32, Float64, N0f8, N0f16)
for C in testcolors, T in testtypes
    box = BoxROI(img .|> C{T}, [125, 125, 175, 175])
    ...
end
```

If the function don't work for `N0f8` type, manually convert it to float type in your src codes and document it (or throw an error as early as you can), otherwise, they're annoying here and there from my personal experiences.

Reply:
ok